### DISCUSSION TITLE
[main] 在只有节点，没有时间线为空时，会导致崩溃

### DISCUSSION THREAD

**Discussion by pizeroLOL** [Class-Widgets/Class-Widgets#177] [Q&A问答] [answered]
https://github.com/Class-Widgets/Class-Widgets/blob/f610252f2de52f1ea4e3d9f0fec3951f1d13b945/main.py#L118-L121

这里是不是应该 return 一个默认值，比如 any 为 False 的时候返回 c_time, 0

**Comment by RinLit-233-shiroko** [comment, marked as answer]
ok已修复 801534b26129038acb61af8e8f773cc014b4c9bd （对了请把Bug之类的东西提交到Issue中）
